ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -5455,3 +5455,77 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+AHU_DFSS_DFVSC_ECOND_CHWDC_DTM_OA_RDC_RHM_RCO2M:
+  guid: "8ef1a26a-fe8f-468a-ba96-ffd0f987c043"
+  description: "AHU with discharge air/fan, chilled water valve operation, humidity monitoring with dual dampers monitoring and control ."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DFVSC
+  - ECOND
+  - CHWDC
+  - DTM
+  - OA
+  - RDC
+  - RTM
+  - RCO2M
+
+AHU_DFVSC_ECOND_HWDC_CHWDC_RDC_RCO2M_RTM_UV:
+  guid: "005172cf-d29a-4c10-94d2-2a5fb6fad51f"
+  description: "AHU with discharge air/fan, UV, chilled water valve and return air temperature monitoring with dual dampers monitoring."
```

Comment:
You mention dual dampers but I only see the one RDC type. Is the other one the outside air damper in ECOND? Please clarify. Please also add the heating water into the description.

Reply:
Yes the other damper is referring to the outside air damper in ECOND